docs/configuration/sorting.rst

```diff
@@ -0,0 +1,59 @@
+Sorting
+=======
+
+CrudJsonApi fully supports
+`JSON API Sorting <http://jsonapi.org/format/#fetching-sorting>`_
+which allows you to sort the results produced by your API according to one
+by passing one or more criteria to your request using the ``sort`` parameter.
+
+Before continuing please note that the default sort order for each field is ascending
+UNLESS the field is prefixed with a hyphen (``-``) in which case the sort order will
+be descending.
+
+Single Field Sorting
+^^^^^^^^^^^^^^^^^^^^
+
+To select all currencies and sort the results by the `code` field:
+
+``/currencies?sort=code``
+
+To select all currencies and sort the results by the `code` field in descending order:
+
+``/currencies?sort=-code``
+
+Multi Field Sorting
+^^^^^^^^^^^^^^^^^^^
+
+@rchavik: I THINK WE NEED AN EXAMPLE OR TESTCASE HERE, SEEMS MISSING. E.G.
+
+- ``/currencies?sort=code,name``
+- ``/currencies?sort=-code,name``
+- ``/currencies?sort=-code,-name``
+
```

Comment:
Are you referring to sorting by two fields of the same table?

Reply:
correct, it looks like "sorting by two fields in same table" is tested, but only for `included` data. If I am missing something please let me know.